File: week3/Constructor.cpp
#include<iostream>
using namespace std;

class Student
{
    public:
        Student()
        {
            string collegeName = "MVGR";
            int collegeCode=33;
            cout<<"CollegeName : "<<collegeName<<endl;
            cout<<"CollegeCode: "<<collegeCode<<endl;
        }
        Student(string fullName, double semPerentage)
        {
            cout<<"FullName : "<<fullName<<endl;
            cout<<"SemPercentage : "<<semPerentage<<endl;
        }
        ~Student(){}
};

int main()
{
    Student venkat;
    Student venkat01("SaiVenkat",86.789);
    return 0;
}

File: week4/Constructor.java
import java.util.Scanner;
class Student
{
  public
    Student()
    {
      String collegeName="MVGR";
      int collegeCode=33;
      System.out.print("CollegeName : "+collegeName+"\n"+"CollegeCode : "+collegeCode+"\n");
    }
    Student(String fullName,double semPercentage)
    {
      System.out.print("FullName : "+fullName+"\n"+"SemPercentage : "+semPercentage);
    }
}

public class Constructor
{
  public static void main(String[] args)
  {
    @SuppressWarnings("unused")
    Student venkat=new Student();
    Scanner input=new Scanner(System.in);
    System.out.print("FullName : ");
    String fullName=input.nextLine();
    System.out.print("SemPercentage : ");
    double semPercentage=input.nextDouble();
    input.close();
    @SuppressWarnings("unused")
    Student venkat01=new Student(fullName,semPercentage);

  }
}
